canvas_modules/common-canvas/src/common-canvas/cc-panels.jsx

```diff
@@ -138,7 +138,7 @@ class CommonCanvasPanels extends React.Component {
 				containingDivId={this.props.containingDivId}
 			/>
 		);
-		const rightFlyout = (<CommonCanvasRightFlyout />);
+		const rightFlyout = (<CommonCanvasRightFlyout containingDivId={this.props.containingDivId} canvasController={this.props.canvasController} />);
```

Comment:
Can't this code be structured the same way as we do it for left flyout below? That is, package it in a function; return null if the panel is not open etc.  I'd like the code that handles the left flyout and the code that handles the right flyout be as much as possible structured in the same way.

Reply:
Updated to include `isRightPanelOpen` also updated `common-canvas-test.js` as the condition `{ showRightFlyout: false }` was failing because of the change.